chain/ethereum/src/ethereum_adapter.rs

```diff
@@ -1300,19 +1300,7 @@ impl EthereumAdapterTrait for EthereumAdapter {
                             call.block_ptr.clone(),
                             call.gas,
                         )
-                        .map(move |result| {
-                            // Don't block handler execution on writing to the cache.
-                            let for_cache = result.0.clone();
-                            let _ = graph::spawn_blocking_allow_panic(move || {
-                                cache
-                                    .set_call(call.address, &call_data, call.block_ptr, &for_cache)
-                                    .map_err(|e| {
-                                        error!(logger, "call cache set error";
-                                                   "error" => e.to_string())
-                                    })
-                            });
-                            result.0
-                        }),
+                        .map(move |result| result.0),
```

Comment:
I feel like this could take a bit of refactoring, but I'm not sure it's possible to make it more nice looking